about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <div class="top-head">
+        <ul class="top-desk">
+        <li><img class="fb" src="asset/facebook.png" alt="facebook"></li>
+        <li><img class="twit" src="asset/twitter.png" alt="twitter"></li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Log out</li>
+        </ul>
+    </div>
+    <div class="desk-menu">
+        <img src="asset/cc global logo.png">
+        <ul class="desk-list">
+            <li onclick="openmenu()" class="ab-desk"><a href="./index.html">Home</a></li>
+            <li onclick="openmenu()" class=""><a href="#pro">Program</a></li>
+            <li onclick="openmenu()"><a href="#"></a>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li class="desk-cc">CC Campaign</li>
+        </ul>
+    </div>
+    <div class="toggle">
+        <img onclick="openMenu()" id="hamburger"  src="asset/mobile_menu 1.png">
+        <img src="asset/cc global logo.png">
+    </div>
+    <div class="menu-mob hidden" id="mobile-menu">
+        <img  onclick="closeMenu()" class="closing" src="asset/Icon - Cancel.png" id="closing-icon">
+
+        <ul class="menu-list" id="hum-list">
```

Comment:
- [ ] kindly change this red color to \#EC5242 so it matches the design
- [ ] please ensure the image, title and text are aligned same in all cards like in the original design. nice work
yours
![Screenshot (1297)](https://github.com/Surafels/Capstone_Project/assets/65298048/5e58ec5b-6b42-4e81-9962-1b9fd57cf5b4)
original

![Screenshot (1299)](https://github.com/Surafels/Capstone_Project/assets/65298048/7a75e1be-8e62-42f3-8b7c-eb920768aaa0)